src/scrapers/max.ts

```diff
@@ -165,6 +167,18 @@ function getChargedCurrency(currencyId: number | null) {
   }
 }
 
+function getMemo(rawTransaction: ScrapedTransaction) {
+  let memo = rawTransaction.comments;
+  if (rawTransaction.fundsTransferReceiverOrTransfer) {
+    memo += ` ${rawTransaction.fundsTransferReceiverOrTransfer}`;
+    if (rawTransaction.fundsTransferComment) {
+      memo += `: ${rawTransaction.fundsTransferComment}`;
+    }
+  }
+
+  return memo;
+}
```

Comment:
What if the `rawTransaction.comments` is empty? This line will create a memo with a space as the first character:

```javascript
memo += ` ${rawTransaction.fundsTransferReceiverOrTransfer}`
```

And from your code I understand that the `rawTransaction.fundsTransferComment` comes always with `rawTransaction.fundsTransferReceiverOrTransfer`.

What do you think about adding a Unit Test for this function, with (as far as you can) real examples, so we will see exactly how the transaction is looks like and how the memo is?

Reply:
Nice catch - comments are empty for fund transfers.
Although it will break existing moneyman hashes, i think it's the right thing to do and accept the risk.

Added some tests as well